simphera-instances.tf

```diff
@@ -1,29 +1,29 @@
-module "simphera_instance" {
```

Comment:
Uncomment this file. This module can be disabled from terraform.tfvars, input variables.